WcaOnRails/config/locales/es.yml

```diff
@@ -2070,7 +2102,7 @@ es:
           el %{end_date_and_time}.
         #original_hash: 7ac8349
         range_past_html: >-
-          El registro en línea se abrió el %{start_date_and_time} y se cerró el
+          El registro en línea se abrió el %{start_date_and_time} y se cerró el 
```

Comment:
@jonatanklosko hum weird that it's including a new space here!
The same happened [here](https://github.com/thewca/worldcubeassociation.org/pull/4422/files#diff-5ccddb4fcb0b8832d7b1397a6fab8617R3613).

@moralsh Just to check, did you touch that translation?

Reply:
Ahhhhh, the same case as [here](https://github.com/thewca/worldcubeassociation.org/pull/4406#discussion_r310409190). I checked and there are also 2 spaces for this phrase. Still no idea how this might've happened if not manually.

Reply:
@viroulep I'm 100% sure I didn't update that key on purpose, wether it was my fault or something else's I'm not sure 😛. New PR is #4427